scan/cli.go

```diff
@@ -162,8 +175,13 @@ func AuditCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	detectedTechnologiesString := coreutils.DetectedTechnologiesToString(detectedTechnologies)
+	if detectedTechnologiesString == "" {
+		log.Info("No technology was detected.")
```

Comment:
No technology was detected.
-->
Could not determine the package manager / build tool used by this project.